model/trace/llm.yaml

```diff
@@ -0,0 +1,190 @@
+          fine-tuned custom model, the value should have a more specific name than the base model that's been fine-tuned.
+      - ref: llm.response.finish_reason
+        requirement_level: recommended
+      - ref: llm.usage.prompt_tokens
+        requirement_level: recommended
+      - ref: llm.usage.completion_tokens
+        requirement_level: recommended
+      - ref: llm.usage.total_tokens
+        requirement_level: recommended
+    events:
+      - llm.content.prompt
+      - llm.content.completion
+
+  - id: llm.content.prompt
+    name: llm.content.prompt
+    type: event
+    brief: >
+      In the lifetime of an LLM span, events for prompts sent and completions received
+      may be created, depending on the configuration of the instrumentation.
+    attributes:
+      - ref: llm.prompt
+        requirement_level: recommended
+        note: >
+          The full prompt string sent to an LLM in a request. If the LLM accepts a more
+          complex input like a JSON object, this field is blank, and the response is
+          instead captured in an event determined by the specific LLM technology semantic convention.
+
+  - id: llm.content.completion
+    name: llm.content.completion
+    type: event
+    brief: >
+      In the lifetime of an LLM span, events for prompts sent and completions received may be created, depending on the configuration of the instrumentation.
+    attributes:
+      - ref: llm.completion
+        requirement_level: recommended
+        note: >
+          The full response string from an LLM. If the LLM responds with a more
+          complex output like a JSON object made up of several pieces (such as OpenAI's message choices),
+          this field is the content of the response. If the LLM produces multiple responses, then this
+          field is left blank, and each response is instead captured in an event determined by the specific
+          LLM technology semantic convention.
+
+  - id: llm.openai
+    type: span
+    brief: >
+      A span representing a request to OpenAI's API, providing additional information on top of the generic llm.request.
+    attributes:
+      - ref: llm.system
+        requirement_level: recommended
+        examples: ['openai', 'microsoft']
+        tag: tech-specific-openai-request
+      - ref: llm.request.model
+        requirement_level: required
+        note: >
+            The name of the LLM a request is being made to. If the LLM is supplied by a
+            vendor, then the value must be the exact name of the model requested. If the
+            LLM is a fine-tuned custom model, the value should have a more specific name
+            than the base model that's been fine-tuned.
+        tag: tech-specific-openai-request
+      - ref: llm.request.max_tokens
+        tag: tech-specific-openai-request
+      - ref: llm.request.temperature
+        tag: tech-specific-openai-request
```

Comment:
What is the default requirement level if not specified?

Reply:
Default is Recommended.